### PR TITLE
add asyncnsq to client libraries

### PR DESCRIPTION
**Pull request by aohan237**
add asyncnsq client to client libraries

### CODE REVIEW

File: _posts/2013-02-01-client_libraries.md [hunk 1]
```diff
@@ -439,6 +439,19 @@ production? Tell us about it on the [mailing list][mailing_list] or Twitter [@im
     <td></td>
     <td></td>
   </tr>  
+  <tr class="warning">
+    <td><a href="https://github.com/aohan237/asyncnsq">asyncnsq</a></td>
+    <td>Python/Http</td>
```
Comment (ploxiln) [inline _posts/2013-02-01-client_libraries.md]: The "/Http" part does not really make sense ... I think you mean "Python/asyncio" but we don't specify "framework" in this column for any other clients, so I'd put just "Python" here.

File: _posts/2013-02-01-client_libraries.md [hunk 2]
```diff
@@ -439,6 +439,19 @@ production? Tell us about it on the [mailing list][mailing_list] or Twitter [@im
...
+    <td><i class="fa fa-check"></i></td>
+    <td><i class="fa fa-check"></i></td>
+    <td></td>
```
Comment (ploxiln) [inline _posts/2013-02-01-client_libraries.md]: asyncnsq supports `lookupd_http_addresses` so I think it more or less does support Discovery?

File: _posts/2013-02-01-client_libraries.md (hunk 1 above)
Comment (aohan237) [reply _posts/2013-02-01-client_libraries.md]: ok..thanks. Python is ok..
in fact, I mean this lib have a asyncio http client.  it add supports for nsq http api

File: _posts/2013-02-01-client_libraries.md (hunk 2 above)
Comment (aohan237) [reply _posts/2013-02-01-client_libraries.md]: yea..

i just dont know how to judge support discovery ,so just leave it...

**Comment by aohan237** [comment]
do i have to modify the code as you said  ? 

or you will take care of it?

**Comment by ploxiln** [comment]
If you could push up another commit with that little change, that would be great.

(For bonus points, you can squash your commits together, and then force-push the branch :)

**Pull request merged**